panther_analysis_tool/main.py

```diff
@@ -398,6 +403,80 @@ def upload_analysis(args: argparse.Namespace) -> Tuple[int, str]:
     return 0, ""
 
 
+def parse_lookup_table(args: argparse.Namespace) -> dict:
+    """Validates and parses a Lookup Table spec file
+
+    Returns a dict representing the Lookup Table
+
+    Args:
+        args: The populated Argparse namespace with parsed command-line arguments.
+
+    Returns:
+        A dict representing the Lookup Table, empty when parsing fails
+    """
+
+    logging.info("Parsing the Lookup Table spec defined in %s", args.path)
+    with open(args.path, "r") as input_file:
+        try:
+            yaml = YAML(typ="safe")
+            lookup_spec = yaml.load(input_file)
+            logging.info("Successfully parse the Lookup Table file %s", args.path)
+        except (YAMLParser.ParserError, YAMLScanner.ScannerError) as err:
+            logging.error("Failed to parse the Lookup Table spec file %s", input_file)
+            logging.error(err)
+            return {}
+        try:
+            LOOKUP_TABLE_SCHEMA.validate(lookup_spec)
+            logging.info("Successfully validated the Lookup Table file %s", args.path)
```

Comment:
Maybe use debug level?

Reply:
I would say no, I usually lean towards verbosity with tools like PAT which run in CI/CD pipelines since it makes debugging easier and this is not super verbose in my view.

Reply:
Sure, makes sense.